dns/Makefile

```diff
@@ -63,3 +68,8 @@ push-local:
 .PHONY: dry-run-local
 dry-run-local: ARGS := --dry-run
 dry-run-local: push-local
+
+precook-configs:
+	source ./lib.sh; \
```

Comment:
I have to admit, I have never in my life `source`d a shell script from within a `make` rule.  It never would have crossed my mind.  You win, Bart.